src/Components/StepThree/StepThree.js

```diff
@@ -160,21 +178,31 @@ const StepThree = ({prevStep, setData, data, submitForm, step, getRealDate}) =>
                         id="doesHaveAgricultureSkillsNo"
                         value="" 
                         name="doesHaveAgricultureSkills"
-                        onChange={handleChangeRadio}/>
+                        onChange={(event) => {
```

Comment:
The same code as on the lines 167-171. We can create a separate function and just pass it here.

Reply:
Even better - we are passing a whole event as payload - in multiple places - and we can use that to simplify the logic.
There can be an ordinary function that accepts event as argument, and calls a dispatch by passing event.target.name as a type, and event.target.value as a payload. in the reducer, then we don't need to access event.target.value - we just take the action.payload. :)